marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/MintTokenEvent.hs

```diff
@@ -123,6 +144,25 @@ tests =
             "propRunnerDoesntTrackUnselectedAssetId"
             propRunnerDoesntTrackUnselectedAssetId
         ]
+    , testGroup
+        "Stability"
+        [ testPropertyNamed
+            "withStability always stable when SecurityParam is 0"
+            "withStabilityAllStable"
+            withStabilityAllStable
```

Comment:
Small convention. Property tests should start with `prop` such as `propWithStabilityAllStable`.